Btmkpl postalcodes

Alex-Quinn opened the pull request:
1. Faker is missing postal code generation for some locales. In particular this PR adds postal code formats for the following locales:
   
   ```
   ne_BE: ####
   fr_LU: ####
   dk_DK: ####
   ```
2. The `no_NO` postalcode was being generated as the literal `####`. This existence of the postalcode class method caused this behavior and was unnecessary. I removed the method and added tests.
3. `sv_SE` postalcodes were missing a space in the format which I added. The format is now `### ##`.
4. Do not include a space in French postcodes. (`#####` instead of `### ##`)
5. The tests for the `ne_NP` locale were failing due to not handling logic for 3 part names. I fixed that issue.

@chrissiedeist 
# Notes

There are more locales with missing or wrong postalcodes but we are punting on those for now. I'll triage those issues as they come up into JIRA issues.

braintreeps committed:
BTMKPL - Generate postalcodes based on correct format

sv_SE: ### ##
ne_BE: ####
fr_LU: ####
no_NO: ####
dk_DK: ####

braintreeps committed:
BTMKPL - Handle name prefixes in  ne_NP name test

cmonty commented on faker/tests/ne_np/__init__.py:
```diff
@@ -37,7 +37,12 @@ def test_names(self):
+      if len(name.split()) == 2:
+          first_name, last_name = name.split()
+      else:
+          prefix, first_name, last_name = name.split()
```
What's with this file changing?

Alex-Quinn replied:
This is the 5th point in the PR description.

> The tests for the ne_NP locale were failing due to not handling logic for 3 part names. I fixed that issue.

cmonty commented:
It looks like the existing Faker formats use `[]` instead of `()`. I think we should be consistent.

Alex-Quinn commented:
> It looks like the existing Faker formats use [] instead of (). I think we should be consistent.

@cmonty this is an inconsistency in the code. Sometimes [`()` is used](https://github.com/braintreeps/faker/blob/master/faker/providers/address/el_GR/__init__.py#L72-L78) and other times [`[]` is used](https://github.com/braintreeps/faker/blob/master/faker/providers/address/no_NO/__init__.py#L29). From what I can tell `()` is used more frequently.

braintreeps committed:
BTMKPL - fr_FR postcodes do not contain space

French postcodes do not contain a space between the third and fouth
digits. Limit the postcode formats for that locale #####.

rchopra commented:
:coffee:

mjrossi commented:
🚆
